src/app/app.ts

```diff
@@ -150,6 +150,51 @@ export function view(state: IState, actions: any) {
       id: 'app',
       class: classNames({'theme-dark': state.settings.darkTheme}),
       oncreate: (el: El) => actions.onCreate(el),
+      ondragover: (ev: Event) => {
+        ev.preventDefault()
+        ev.stopPropagation()
+      },
+      ondragleave: (ev: Event) => {
+        ev.preventDefault()
+        ev.stopPropagation()
+      },
+      ondrop: (ev: any, node: IGraphNodeData) => {
+        ev.preventDefault()
+        ev.stopPropagation()
+        for (const f of ev.dataTransfer.files) {
+          logger.debug('Dropped file path = ' + f.path)
+          if (f.path.includes('.txt')) {
+            const reader = new FileReader()
+            reader.readAsDataURL(f)
+            reader.onloadend = () => {
+              const base64Data: string = reader.result!.toString().split(',')[1]
+              const fileContent = atob(base64Data)
+              actions.graph.createNewNode({
+                position: {x: ev.screenX, y: ev.screenY},
+                parent: null,
+                type: NoteDataType.TEXT,
+                newNodeCallback: (nodeId: GraphNodeId) => {
+                  writeNote(nodeId, NoteDataType.TEXT, fileContent)
+                },
+              })
+            }
+          } else if (f.path.includes('.jpg') || f.path.includes('.png')) {
```

Comment:
There appears to be a problem with importing JPEG's. Can you change this so it only allows you to import PNGs.